package-overrides/github/tinymce/tinymce-dist@4.0.0.json

```diff
@@ -0,0 +1,258 @@
+{
+  "format": "global",
+  "main": "tinymce",
+  "shim": {
+    "tinymce.min": {
+      "exports": "tinymce"
+    },
+    "tinymce": {
+      "exports": "tinymce"
+    },
+    "plugins/advlist/plugin": {
+      "imports": "tinymce",
+      "deps": [
+        "tinymce"
```

Comment:
Does this definitely work? It may be necessary to do backtracking here - `../../tinymce`. Deps are written as if it was a require from within the module file.